src/pro/blocks/modal/block.json

```diff
@@ -0,0 +1,150 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "themeisle-blocks/modal",
+	"title": "Modal",
+	"category": "themeisle-blocks",
+	"description": "Display your content in beautiful Modal with many customization options. Powered by Otter.",
+	"keywords": [ "modal", "lightbox" ],
+	"textdomain": "otter-blocks",
+	"attributes": {
+		"id": {
+			"type": "string"
+		},
+		"minWidth": {
+			"type": ["number", "string"]
+		},
+		"maxWidth": {
+			"type": ["number", "string"]
+		},
+		"trigger": {
```

Comment:
Why do we have these unused attributes from the Popup block?

Reply:
I kept it for compatibility. Should I also transform from Popup to Modal and vice versa?

Reply:
Yea, you can add transformation but just remove the trigger as both are separate blocks and this attribute doesn't do anything in modal.